src/crate/crash/command.py

```diff
@@ -494,9 +509,42 @@ def main():
 
     crate_hosts = [host_and_port(h) for h in args.hosts]
     error_trace = args.verbose > 0
+
+    force_passwd_prompt = args.password
+    username = args.username
+    password = None
+
+    # If no user provided check env. variables LOGNAME, USER, LNAME and USERNAME
+    if not username:
+        username = getuser()
```

Comment:
If we support this implicit user retrieval I think we also have to document it. Could also be changed to `username = args.username or getuser()` in the definition further above.